Modules/GLO/src/ITSTPCMatchingTask.cxx

```diff
@@ -99,72 +106,7 @@ void ITSTPCMatchingTask::initialize(o2::framework::InitContext& /*ctx*/)
 
   mMatchITSTPCQC.initDataRequest();
   mMatchITSTPCQC.init();
-
```

Comment:
@chiarazampolli I changed the way we publish the histograms so that we don't need to duplicate the PRs. I can avoid it if you think it's unecessary